src/main/resources/build-dev.template.properties

```diff
@@ -14,6 +14,13 @@ app.frontend.url = http\://localhost:4200
 # This is the port to connect with local Datastore emulator.
 app.localdatastore.port = 8484
 
+# Configurations for postgres
```

Comment:
Maybe can add more explanation here on what values these fields should be set to (e.g. These correspond to the postgres instance details in the docker image and no change required etc)